recipes/behaviortree.cpp/all/conanfile.py

```diff
@@ -169,12 +89,8 @@ def validate(self):
         if self.settings.compiler == "clang" and str(self.settings .compiler.libcxx) == "libstdc++":
             raise ConanInvalidConfiguration(f"{self.ref} needs recent libstdc++ with charconv. please switch to gcc, or to libc++")
 
-        if self.settings.compiler == "apple-clang" and cross_building(self) and self.settings.arch in ["x86", "x86_64"]:
```

Comment:
It looks like this was outdated.. I removed it just to see what happens and CICD didn't complain so we should be good?

Reply:
Totally fine!